yarn.lock

```diff
@@ -5626,11 +5616,16 @@
   resolved "https://registry.yarnpkg.com/@types/he/-/he-1.1.1.tgz#19e14033c4ee8f1a702c74dcc6182664839ac2b7"
   integrity sha512-jpzrsR1ns0n3kyWt92QfOUQhIuJGQ9+QGa7M62rO6toe98woQjnsnzjdMtsQXCdvjjmqjS2ZBCC7xKw0cdzU+Q==
 
-"@types/history@*", "@types/history@^4.7.3":
+"@types/history@*":
   version "4.7.3"
   resolved "https://registry.yarnpkg.com/@types/history/-/history-4.7.3.tgz#856c99cdc1551d22c22b18b5402719affec9839a"
   integrity sha512-cS5owqtwzLN5kY+l+KgKdRJ/Cee8tlmQoGQuIE9tWnSmS3JMKzmxo2HIAk2wODMifGwO20d62xZQLYz+RLfXmw==
 
+"@types/history@^4.7.9":
```

Comment:
These two versions should be merged so that we only have 4.7.9.

Reply:
I tried this with 

```diff
diff --git a/yarn.lock b/yarn.lock
index a576f77946c..7f717a21f55 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -5616,12 +5616,7 @@
   resolved "https://registry.yarnpkg.com/@types/he/-/he-1.1.1.tgz#19e14033c4ee8f1a702c74dcc6182664839ac2b7"
   integrity sha512-jpzrsR1ns0n3kyWt92QfOUQhIuJGQ9+QGa7M62rO6toe98woQjnsnzjdMtsQXCdvjjmqjS2ZBCC7xKw0cdzU+Q==
 
-"@types/history@*":
-  version "4.7.3"
-  resolved "https://registry.yarnpkg.com/@types/history/-/history-4.7.3.tgz#856c99cdc1551d22c22b18b5402719affec9839a"
-  integrity sha512-cS5owqtwzLN5kY+l+KgKdRJ/Cee8tlmQoGQuIE9tWnSmS3JMKzmxo2HIAk2wODMifGwO20d62xZQLYz+RLfXmw==
-
-"@types/history@^4.7.9":
+"@types/history@*", "@types/history@^4.7.9":
   version "4.7.9"
   resolved "https://registry.yarnpkg.com/@types/history/-/history-4.7.9.tgz#1cfb6d60ef3822c589f18e70f8b12f9a28ce8724"
   integrity sha512-MUc6zSmU3tEVnkQ78q0peeEjKWPUADMlC/t++2bI8WnAG2tvYRPIgHG8lWkXwqc8MsUF6Z2MOf+Mh5sazOmhiQ==
```

But that resulted in type errors in Canvas:

```
ERROR Type check failed in x-pack/plugins/canvas/tsconfig.json:
ERROR x-pack/plugins/canvas/public/routes/workpad/hooks/use_restore_history.ts:22:52 - error TS2769: No overload matches this call.
        Overload 1 of 3, '(initialValue: string | undefined): MutableRefObject<string | undefined>', gave the following error.
          Argument of type 'unknown' is not assignable to parameter of type 'string | undefined'.
            Type 'unknown' is not assignable to type 'string'.
        Overload 2 of 3, '(initialValue: string | null | undefined): RefObject<string | undefined>', gave the following error.
          Argument of type 'unknown' is not assignable to parameter of type 'string | null | undefined'.
            Type 'unknown' is not assignable to type 'string'.

      22   const previousState = useRef<string | undefined>(historyState);
                                                            ~~~~~~~~~~~~


      x-pack/plugins/canvas/public/routes/workpad/hooks/use_restore_history.ts:28:7 - error TS2322: Type 'unknown' is not assignable to type 'string | undefined'.
        Type 'unknown' is not assignable to type 'string'.

      28       previousState.current = historyState;
               ~~~~~~~~~~~~~~~~~~~~~

      x-pack/plugins/canvas/public/routes/workpad/hooks/use_restore_history.ts:29:38 - error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'string'.

      29       dispatch(restoreHistory(decode(historyState)));
                                              ~~~~~~~~~~~~

      x-pack/plugins/canvas/public/routes/workpad/hooks/use_restore_history.ts:33:5 - error TS2322: Type 'unknown' is not assignable to type 'string | undefined'.
        Type 'unknown' is not assignable to type 'string'.

      33     previousState.current = historyState;
             ~~~~~~~~~~~~~~~~~~~~~

      x-pack/plugins/canvas/public/routes/workpad/hooks/use_workpad_history.ts:24:49 - error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'string'.

      24     const decodedState = locationState ? decode(locationState) : {};
                                                         ~~~~~~~~~~~~~


      Found 5 errors.
```

Reply:
Odd 🤷 If it gives problems just leave it as is I guess

Reply:
Yeah, I might need to follow-up with it after this PR.